### ISSUE TITLE
Not working with Seamonkey 2.40

### ISSUE THREAD

**Issue by Olifair** [poohsen/thunderlink#33]
Hello, does anybody know why Thunderlink is not working with Seamonkey Mail V 2.40 ? 
With not working I mean that when opening the xpi file with Seamonkey, nothing happens and the plugin is not installed. 
Can this be forced ? I would love to be able to make links from Freeplane to emails in Thunderbird. 

Cheers
Oliver

**Comment by poohsen** [comment]
Did it work with earlier versions?

**Comment by Olifair** [comment]
No idea. I have just tried to install with this version.

**Comment by poohsen** [comment]
Ah. I hope it did not pop up in some SeaMonkey add-on repository then? it was never written with that in mind. Nor tested with it...

**Comment by Olifair** [comment]
No, it came up on the general mozilla plugin page. There was no direct comment that it would not work and actually it did not block before downloading (as can happen with other pl7gin which are not for your program version), but only during the install I get the message.
Before I will switch from Seamonkey to Firefox/ Thunderbird for this plugin a quick question : is this plugin still actively maintained, meaning is it be kept working through version  changes of thunderbird ? 
Cheers
Oliver

**Comment by poohsen** [comment]
no, it's not. sorry.
